integration/integrate-with-splunk.adoc

```diff
@@ -0,0 +1,52 @@
+[id="integrate-with-splunk"]
+= Integrating with Splunk
+include::modules/common-attributes.adoc[]
+:context: integrate-with-splunk
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://www.splunk.com/[Splunk], you can forward alerts from {product-title} to Splunk and view vulnerability and compliance related data from within Splunk.
```

Comment:
Shouldn't it be link:https://www.splunk.com/[Splunk]? The link works, but we add the `link` before the URL: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Reply:
Thank you @stevsmit I have updated this one, and a bunch of others.